ramls/types/collection-item.raml

```diff
@@ -0,0 +1,16 @@
+collection-item:
+  description: Entity representing a <<resourcePathName|!singularize>>
```

Comment:
So where does this `resourcePathName` come from in these resourceType files? Does it just use the string in the URL that this gets attached to via `type` above?

Reply:
thats my understanding but I am modifying this back to following 100 Tutorial, the PR for which I will push by EOD. I think this reusable way they suggest is purely for RESTful APIs where the data matches exactly between GET and POSTs, which is not our case. We'll see which way to proceed after we have both to compare.